NewBank/newbank/server/Customer.java

```diff
@@ -3,39 +3,50 @@
 import java.util.ArrayList;
 
 public class Customer {
-	
-	private ArrayList<Account> accounts;
-	
-	public Customer() {
-		accounts = new ArrayList<>();
-	}
-	
-	public String accountsToString() {
-		String s = "";
-		for(Account a : accounts) {
-			s += a.toString();
-		}
-		return s;
-	}
-
-	public boolean addAccount(Account account) {
-		for (Account a: accounts){
-			if (a.getName().equals(account.getName())){
-				return false;
-			}
-		}
-		accounts.add(account);
-		return true;
-	}
-	public ArrayList<Account> getAccountList() { return accounts; }
-	public Boolean move(Account fromAccount, Account toAccount, double amount){
-		//check the <From> account has enough money
-		if (fromAccount.getBalance() >= amount) {
-			//update balance
-			fromAccount.removeMoney(amount);
-			toAccount.addMoney(amount);
-			return true;
-		}
-	return false;
-	}
+
+    private ArrayList<Account> accounts;
+
+    private String password;
```

Comment:
Sorry for adding a lot of white spaces changes. IntelliJ doesn't like tabs, I don't know how to force it not to change tabs to spaces. 

I'll mark all significant changes in this file. This is one of them.